### PR TITLE
Fix SQPNP

### PR DESCRIPTION
**Pull request by JoshuaAN**

### CODE REVIEW

File: src/main/cpp/Robot.cpp [hunk 1]
```diff
@@ -25,7 +25,7 @@ Robot::Robot() : frc::TimesliceRobot{2_ms, 5_ms} {
   if constexpr (RobotBase::IsSimulation()) {
     // auto inst = nt::NetworkTableInstance::GetDefault();
     // inst.StopServer();
-    // inst.SetServer("localhost");
+    // inst.SetServer("192.168.1.152");
```
Comment (jlbabilino) [inline src/main/cpp/Robot.cpp]: Where did this IP come from?

File: src/main/include/subsystems/Vision.hpp [hunk 2]
```diff
@@ -28,15 +28,20 @@ class Vision : public frc2::SubsystemBase {
 
                                 735.0038676061282,
                             0.0, 666.4393853583332, 0.0, 734.0230911175827,
-                            388.3489253256961, 0.0, 0.0, 1.0);
+                            388.3489253256961, 0.0, 0.0, 1.0
```
Comment (jlbabilino) [inline src/main/include/subsystems/Vision.hpp]: Could you add some comments documenting which constants are for which camera (on your Mac, the global shutter one, etc.).

File: simgui.json [hunk 3]
```diff
@@ -116,14 +112,18 @@
...
+          "open": true
         },
         "oldpv": {
           "open": true
         },
         "open": true
       },
       "photonvision": {
+        "Integrated_Webcam": {
+          "open": true
```
Comment (jlbabilino) [inline simgui.json:125]: Is this so you can see the camera feed on Glass?

**Comment by jlbabilino** [comment]
Please fix formatting/lint

File: simgui.json (hunk 3 above)
Comment (JoshuaAN) [reply simgui.json:125]: Yes

File: src/main/include/subsystems/Vision.hpp (hunk 2 above)
Comment (JoshuaAN) [reply src/main/include/subsystems/Vision.hpp]: Done

File: src/main/cpp/Robot.cpp (hunk 1 above)
Comment (JoshuaAN) [reply src/main/cpp/Robot.cpp]: Changed back to localhost

**Review by jlbabilino — APPROVED** [review]

**Pull request merged**